DigitalExcellence/dex-frontend
# Feature: sentry error

## Description
Implemented global error logging with the use of Sentry.

## Type of change


## Checklist


## Steps to Test or Reproduce

For testing locally, environment.ts should be changed in the following ways:
1. Add the following property: `sentryDsnUrl:`, contact me for the actual value
2. Change production from false -> true

Any error being throw now should appear on the dashboard on sentry.io. Access can be request from @Brend-Smits 

## Link to issue

Closes: #77
